<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VariableManage</class>
 <widget class="QWidget" name="VariableManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#VariableManage{
border-image: url(:/widget/调距桨背景.png);
}

QGroupBox{
	background-color: rgba(16, 57, 70, 1);
	border: 0px solid rgb(111, 255, 255);
	margin-top:3.3ex;
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QGroupBox::title{
	subcontrol-origin: margin;
	subcontrol-position: top center;
	background-color: rgba(19, 67, 79, 1);
	padding: 7 1000px;
}

QLineEdit{
	background-color: rgba(255, 255, 255,0.2);
	border-width:0;
	border-style:outset;
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QLabel{
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QRadioButton{
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QRadioButton::indicator{
	border-color: rgba(255, 255, 255,0);
	background:transparent;
}


QRadioButton::indicator:checked{
	border-image:url(:/widget/images/green.png)
}

QRadioButton::indicator:unchecked{
	border-image:url(:/widget/images/unchecked.png)
}

QPushButton{
	border: 2px solid rgba(61, 255, 255,0.4);
	background-color: rgba(12, 34, 45, 0.6);
	font: 12pt &quot;微软雅黑&quot;;
	color: rgba(102, 249, 247, 1);
	text-align：center;
}

QPushButton::disabled{
	border: 0px;
	color: rgba(102, 249, 247, 0.3);
	text-align：center;
}

QPushButton::pressed{
	border: 2px solid rgba(61, 255, 255,0.8);
	background-color: rgba(12, 34, 45, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QPushButton::hover{
	border: 2.5px solid rgba(61, 255, 255,0.8);
	background-color: rgba(12, 34, 45, 0.6);
	font: 75 12pt &quot;微软雅黑&quot;;
}

QDoubleSpinBox{
	border: 0px solid rgba(61, 255, 255,0.4);
	background-color: rgba(255, 255, 255, 0.2);
	font: 12pt &quot;Times New Roman&quot;;
	color: rgba(102, 249, 247, 1);
	text-align：center;
	padding-left:10px;
}


QTabWidget{
	background-color: rgba(19, 67, 79, 1);
	border-color: rgba(19, 67, 79, 1);
	border:0px;
	margin:0px;
}

QTabWidget::pane{
	border:0px;
	margin:0px;
}

QTabWidget::tab-bar {
	left: 2px;
	border:0px;
	margin:0px;
}

QTabBar::tab:first {
	background-color: rgba(11, 49, 61, 1);
	border-color: rgba(11, 49, 61, 1);
	color: rgba(102, 249, 247, 1);
	min-width: 4em;
	min-height:1em;
}
QTabBar::tab {
	font: 12pt &quot;微软雅黑&quot;;
	background-color: rgba(11, 49, 61, 255);
	border-color: rgba(11, 49, 61, 255);
	color: rgba(102, 249, 247, 1);
	min-width: 4em;
	min-height:1em;
}

QTabBar::tab:first:selected {
	min-width: 4em;
	min-height:1em;
}

QTabBar::tab:selected {
margin-bottom:0.2em;
}

QTabBar::tab:!selected{
	min-width: 4em;
	min-height:1em;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <widget class="QLabel" name="label_33">
       <property name="text">
        <string>变量名称：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_VariableName">
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	background-color: rgba(255, 255, 255,1);
	color: rgb(0, 0, 0);
	border-width:0;
	border-style:outset;
	font: 12pt &quot;微软雅黑&quot;;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_35">
       <property name="text">
        <string>变量类型：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Cmb_VariableType">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentText">
        <string>ALL</string>
       </property>
       <item>
        <property name="text">
         <string>ALL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>基础信号</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>中间信号</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>报警信号</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>故障信号</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_VariableBaseSelect">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>模糊查询</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget_VariableBase">
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255,0.2);</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <widget class="QPushButton" name="Btn_VariableBaseNew">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>新增变量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_VariableBaseAlter">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>修改变量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_VariableBaseDelete">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>删除变量</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
